src/components/top-bar/styles.ts

```diff
@@ -0,0 +1,5 @@
+import { styled } from '@linaria/react'
+
+export const ElTopBarLogo = styled.a`
+  padding: var(--spacing-spacing-2, 8px) var(--spacing-spacing-2, 8px) var(--spacing-spacing-2, 8px) 0px;
```

Comment:
@nurm717123 
**issue:** There is a naming convention difference between DS figma semantic variables v/s token file variables in the codebase. There is no var called `--spacing-spacing-2` in the codebase, it is `--spacing-2` , 

**suggestion:** Please review all variables and cross-reference with the token file.

Reply:
as dicussed in the group chat, I will use the `--spacing`, thanks for noticing 👍